cmd/svcat/broker/register_cmd.go

```diff
@@ -0,0 +1,74 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package broker
+
+import (
+	"fmt"
+
+	"github.com/kubernetes-incubator/service-catalog/cmd/svcat/command"
+	"github.com/kubernetes-incubator/service-catalog/cmd/svcat/output"
+	"github.com/spf13/cobra"
+)
+
+type RegisterCmd struct {
+	BrokerName string
+	Context    *command.Context
+	URL        string
+}
+
+// NewRegisterCmd builds a "svcat register" command
+func NewRegisterCmd(cxt *command.Context) *cobra.Command {
+	registerCmd := &RegisterCmd{
+		Context: cxt,
+	}
+	cmd := &cobra.Command{
+		Use:   "register NAME --url URL",
+		Short: "Registers a new broker with service catalog",
+		Example: command.NormalizeExamples(`
+		svcat register mysqlbroker --url http://mysqlbroker.com
```

Comment:
I believe this needs to be `svcat register broker mysqlbroker ...`, so it's aligned with `svcat sync broker mysqlbroker`

Reply:
I'm not sure about that. I would thing `register` would be like `provision` and `bind`, neither of which require you to specify the object types they create. @carolynvs thoughts?

Reply:
I'm with Jonathan on that, for our domain specific verbs that only apply to a single resource type (in this case a broker), I would prefer that we are consistent and not require the redundant noun.